barretenberg/cpp/src/barretenberg/transcript/transcript.hpp

```diff
@@ -23,6 +23,16 @@ class TranscriptManifest {
         std::vector<std::string> challenge_label;
         std::vector<std::pair<std::string, size_t>> entries;
 
+        void print()
```

Comment:
Just reworking this so you can print a single round of the manifest (so that when a transcript test fails you can just look at the failing round)

Reply:
Nicee!